src/lambdas/auditMessageFirehoseReingest/handler.test.ts

```diff
@@ -0,0 +1,252 @@
+import { SQSEvent } from 'aws-lambda'
+import { when } from 'jest-when'
+import { mockLambdaContext } from '../../utils/tests/mockLambdaContext'
+import { testS3TestEvent } from '../../utils/tests/testEvents/testS3SqsEvent'
+import { deleteOrUpdateS3Objects } from './deleteOrUpdateS3Objects'
+import { getAuditEvents } from './getAuditEvents'
+import { handler } from './handler'
+import { sendAuditEventsToFirehose } from './sendAuditEventsToFirehose'
+
+jest.mock('./getAuditEvents', () => ({
+  getAuditEvents: jest.fn()
+}))
+
+jest.mock('./sendAuditEventsToFirehose', () => ({
+  sendAuditEventsToFirehose: jest.fn()
+}))
+
+jest.mock('./deleteOrUpdateS3Objects', () => ({
+  deleteOrUpdateS3Objects: jest.fn()
+}))
+
+describe('handler', () => {
+  const sqsEvent: SQSEvent = {
+    Records: [
+      {
+        body: JSON.stringify({
+          Records: [
+            {
+              s3: {
+                bucket: {
+                  name: 'mockBucket'
+                },
+                object: {
+                  key: 'failures/your-object-key'
+                }
+              }
+            }
+          ]
+        }),
+        messageId: 'messageId1'
+      }
+    ]
+  } as SQSEvent
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('successfully receives an SQS event, and returns an SQS Batch response', async () => {
+    when(getAuditEvents).mockResolvedValue({
+      successfulResults: [
+        {
+          auditEvents: [],
+          bucket: 'mockBucket',
+          key: 'failures/your-object-key',
```

Comment:
A lot of repetition of the same string here, would be better to use a constant to save typos later on

Reply:
Done